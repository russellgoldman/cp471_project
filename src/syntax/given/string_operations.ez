String str3 = str1 + str2;