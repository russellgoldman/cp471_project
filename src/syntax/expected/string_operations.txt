\Tree [.program [.nextLine [.nextLinePrime [.statement
	[.expression [.assignmentExpression
		[.variableDeclaration
			[.String ]
			[.str3 ]
		]
		[.= ]
		[.sumExpression
			[.multiplyExpression [.factor [.mutable [.str1 ] ] ] ]
			[.sumExpressionPrime
				[.sumOperator [.+ ] ]
				[.multiplyExpression [.factor [.mutable [.str2 ] ] ] ]
			]
		]
	] ]
	[.; ]
] ] ] ]